src/test/resources/fastapi/requirements.txt

```diff
@@ -1,2 +1,2 @@
-fastapi[standard]>=0.113.0,<0.114.0
-pydantic>=2.7.0,<3.0.0
+fastapi[standard]==0.115.6
+pydantic==2.4.0
```

Comment:
Why pin to an older version?

Reply:
That was unintended, let me upgrade and try with the newest. It was just Intellij complaining about dependencies.